src/Umbraco.Web.UI.Client/src/views/propertyeditors/mediapicker/mediapicker.controller.js

```diff
@@ -4,23 +4,27 @@ angular.module('umbraco').controller("Umbraco.PropertyEditors.MediaPickerControl
     function ($rootScope, $scope, dialogService, entityResource, mediaResource, mediaHelper, $timeout, userService, $location, localizationService) {
 
         //check the pre-values for multi-picker
-        var multiPicker = $scope.model.config.multiPicker && $scope.model.config.multiPicker !== '0' ? true : false;

```

Comment:
See my other comments - the multiPicker value will still need to be taken into account since that value will still exist for upgraded sites